Android/Java/app/src/main/java/com/levelplay/levelplaydemo/DemoActivityListener.java

```diff
@@ -1,13 +1,13 @@
-package com.ironsource.ironsourcesdkdemo;
+package com.levelplay.levelplaydemo;
 
-import com.ironsource.mediationsdk.model.Placement;
+import com.unity3d.mediation.rewarded.LevelPlayReward;
 
 interface DemoActivityListener {
     void setEnablementForButton(DemoButtonIdentifiers buttonIdentifier, boolean enable);
```

Comment:
there's a warning 'Class 'DemoButtonIdentifiers' is exposed outside its defined visibility scope. Can you change the class to public to remove this warning from people using the demo